test/e2e/framework/helper/validations/certificates.go

```diff
@@ -303,7 +299,13 @@ func ExpectCorrectTrustChain(certificate *cmapi.Certificate, secret *corev1.Secr
 	}
 
 	if _, err := cert.Verify(opts); err != nil {
-		return err
+		return fmt.Errorf(
+			"verify error. CERT:\n%s\nROOTS\n%s\nINTERMEDIATES\n%v\nERROR\n%s\n",
+			pretty.Sprint(cert),
+			pretty.Sprint(rootCertPool),
+			pretty.Sprint(intermediateCertPool),
+			err,
```

Comment:
The verification error wasn't very helpful, so I added this extra information to it.
There's rather a lot of output, but it helps figure out exactly what was in each certificate pool that caused the verfiication to fail.